platform/core-api/core-api.iml

```diff
@@ -22,5 +22,6 @@
     </orderEntry>
     <orderEntry type="library" name="asm" level="project" />
     <orderEntry type="library" name="CGLIB" level="project" />
+    <orderEntry type="library" name="Guava" level="project" />
```

Comment:
Could you please avoid introducing new dependencies unless it is absolutely necessary